src/pdbvangogh/img.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+import numpy as np
+import PIL
+
+
+def tensor_to_image(tensor):
+    tensor = tensor * 255
+    tensor = np.array(tensor, dtype=np.uint8)
+    if np.ndim(tensor) > 3:
+        assert tensor.shape[0] == 1
+        tensor = tensor[0]
+    return PIL.Image.fromarray(tensor)
+
+
+def load_img(path_to_img):
+    max_dim = 512
+    img = tf.io.read_file(path_to_img)
+    img = tf.image.decode_image(img, channels=3)
+    img = tf.image.convert_image_dtype(img, tf.float32)
+
+    shape = tf.cast(tf.shape(img)[:-1], tf.float32)
+    long_dim = max(shape)
+    scale = max_dim / long_dim
+
+    new_shape = tf.cast(shape * scale, tf.int32)
+
+    img = tf.image.resize(img, new_shape)
+    img = img[tf.newaxis, :]
+    return img
+
+
+def imshow(image, title=None):
+    if len(image.shape) > 3:
+        image = tf.squeeze(image, axis=0)
+
+    plt.imshow(image)
+    if title:
+        plt.title(title)
+
+
+def resize_pil(image, new_width):
+    from PIL import Image
```

Comment:
style nit: in general imports should go at the top of the file, if it's needed inline it usually hints at a circular dependency in the code that should be addressed.  In this case though, import not needed?